ports/stm32/powerctrl.c

```diff
@@ -622,6 +697,10 @@ void powerctrl_enter_standby_mode(void) {
     // save RTC interrupts
     uint32_t save_irq_bits = RTC->CR & CR_BITS;
 
+    // disable register write protection
+    RTC->WPR = 0xca;
+    RTC->WPR = 0x53;
```

Comment:
Why is this needed?

Reply:
Because the RTC registers are left write-protected in `pyb_rtc_wakeup`, and these writes in `powerctrl_enter_standby_mode`, useful or not, have no effect:
```
Breakpoint 1, powerctrl_enter_standby_mode () at powerctrl.c:674
674 void powerctrl_enter_standby_mode(void) {
705     RTC->CR &= ~CR_BITS;
(gdb) 
708     RTC->ISR &= ~ISR_BITS;
(gdb) print /x RTC->CR
$18 = 0x44402
(gdb) set RTC->WPR = 0xca
(gdb) set RTC->WPR = 0x53
(gdb) set RTC->CR &= ~CR_BITS
(gdb) print /x RTC->CR
$19 = 0x40402
```

Reply:
Ah, I see!  Yes it's needed.